spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe "Show Page" do 
+  describe "Project show page visit" do 
+    # US 1, 3, ext 1, ext 2 
+    it "shows the prjects theme and material" do 
+
+      recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+      
+      news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+      
+      jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+
+      ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+      ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+
+      visit "/projects/#{news_chic.id}"
+
+      expect(page).to have_content("Project name: #{news_chic.name}")
+      expect(page).to have_content("Material: #{news_chic.material}")
+      expect(page).to have_content("Challenge Theme: #{recycled_material_challenge.theme}")
+
+      expect(page).to have_content("Number of Contestants:")
+      expect(page).to have_content(2)
+
+      expect(page).to have_content("Average Contestant Experience:")
+      expect(page).to have_content(12.5)
+
+      expect(page).to have_content("Add a contestant to this project")
+      expect(page).to have_field("Name")
+      expect(page).to have_button("Add Contestant To Project")
+    end
```

Comment:
Break things up into different tests. 
1. I see that projects name and material
2. I see the theme of the challenge that this project belongs to
3. I see the count of contestants on this project
4. I see the average years experience for the contestants
5. I see a form to add a contestant
6. When I fill out a field with an existing contestants id and hit 'Add Contestant to Project' I'm taken back to the project's show page
7. On the project's show page I see that the number of contestants has increased by 1
8. And on the Contestants Index page I see that project listed under that contestant's name